### PR TITLE
no support urls for TPOs

### PR DESCRIPTION
**Pull request by sarveshpro**
Changes in this pull request:

- [x] remove support URL for TPOs

- [x] change text and add translation resources
image

`Gift to` `Individual Name`
`Plant trees with` `other profile type`

Disable for TPO.

### CODE REVIEW

**Comment by norbertschuler** [comment]
What about this PR - ready to review or do we wait until the second task of the description is done as well?

**Comment by sarveshpro** [comment]
> What about this PR - ready to review or do we wait until the second task of the description is done as well?

It's done, waiting for @sagararyal to approve the textual changes.

**Comment by norbertschuler** [comment]
- [ ] I don't understand how the sentence "Plant trees with other profile type" instead the old text "Select a project" makes sense here:

Bildschirmfoto 2020-12-01 um 09 57 06

- [ ] The box is overlapping with the projects on a smaller screen. If we can not avoid the overlapping, it at least should be the frontmost element. shouldn't it?

Bildschirmfoto 2020-12-01 um 09 57 19

**Comment by sarveshpro** [comment]
> * [x]  I don't understand how the sentence "Plant trees with other profile type" instead the old text "Select a project" makes sense here:
> 
> Bildschirmfoto 2020-12-01 um 09 57 06
> 
> * [x]  The box is overlapping with the projects on a smaller screen. If we can not avoid the overlapping, it at least should be the frontmost element. shouldn't it?
> 
> Bildschirmfoto 2020-12-01 um 09 57 19

Yeah, I misunderstood that one. fixed it.

It's on top now.

**Comment by norbertschuler** [comment]
> Yeah, I misunderstood that one. fixed it.
> 
> It's on top now.

@sarveshpro I am sorry, but it's still looking like this for me trying to support an individual profile like https://planet-webapp-git-feature-no-tpo-support-url.planetapp.vercel.app/t/norbert-schuler

Bildschirmfoto 2020-12-01 um 12 27 20

File: src/features/donations/components/treeDonation/DirectGift.tsx [hunk 1]
```diff
@@ -11,11 +13,16 @@ export default function DirectGift({
...
+  const { t } = useTranslation(['donate', 'common']);
   return (
     <div className={styles.giftContainer}>
       <div className={styles.textContainer}>
-        <div className={styles.giftTo}>Gift to {directGift.displayName}</div>
-        <div className={styles.selectProject}>Select a project</div>
+        <div className={styles.giftTo}>
+          {directGift.type === 'individual'
+            ? t('donate:giftToName', { name: directGift.displayName })
+            : t('donate:plantTreesWith', { name: directGift.displayName })}
```
Comment (norbertschuler) [inline src/features/donations/components/treeDonation/DirectGift.tsx:23]: This is the text for supporting TPOs? But we do not show the support link - so can I test this?

**Review by norbertschuler — APPROVED** [review]

File: src/features/donations/components/treeDonation/DirectGift.tsx (hunk 1 above)
Comment (norbertschuler) [reply src/features/donations/components/treeDonation/DirectGift.tsx:23]: I will test this once it's merged with a educational profile I have created.

**Comment by norbertschuler** [comment]
@sagararyal is this ok for you to merge?

**Comment by sagararyal** [comment]
This is fine to merge however there is a bug on safari; the box becomes narrow. caused by lack of min width I think.

**Pull request merged**